### ISSUE TITLE
Button icon background rotates with loading icon

### ISSUE THREAD

**Issue by vinz243** [Semantic-Org/Semantic-UI#916]
Hi,
My code:

``` html
<div class="ui labeled icon basic small button">
    <i class="loading icon"></i> Please wait...
</div>
```

Result:
![problem](http://i.gyazo.com/1de0e1293406326a63fa4a1f7f5aca85.gif)

How to solve that?

**Comment by domialex** [comment]
This solution doesn't work anymore, the icon is not centered vertically. What would be the correct way to do this?

**Comment by lubber-de** [comment]
This if fixed in the upcoming Version of Fomantic-UI 2.7.2 by  fomantic/Fomantic-UI#361
See http://jsfiddle.net/bztsj9wm/3/
